vars/perfRegressionParallelPipeline.groovy

```diff
@@ -248,7 +254,7 @@ def call(Map pipelineParams) {
                                                         if [[ ! -z "${params.scylla_ami_id}" ]] ; then
                                                             export SCT_AMI_ID_DB_SCYLLA=${params.scylla_ami_id}
                                                         elif [[ ! -z "${supportedVersions}" ]]; then
```

Comment:
in case of supportedVersions you already rewrite params by 
```
 new_params["scylla_version"] = supportedVersions
```

so this elif can be removed and next one updated to:

```
 elif [[ ! -z "${params.scylla_version}" ]] ; then
        export SCT_SCYLLA_VERSION=${new_params.scylla_version}
```

Reply:
it doesn't change anything in the logic, but i rather prefer to be specific about `supportedVersions`, since the function that calculates it, may also return empty for any reason, so we will fail faster